src/types/payments/Pawapay.ts

```
@@ -0,0 +1,166 @@
+  language?: 'EN' | 'FR';
+  country?: string;
+  reason?: string;
+  metadata?: Array<{
+    fieldName: string;
+    fieldValue: string;
+    isPII?: boolean;
+  }>;
+}
+
+export enum PawaPayCorrespondent {
+  MTN_MOMO_BEN = "MTN_MOMO_BEN",
```

ESLint

[error] Strings must use singlequote.

(style/quotes)